### ISSUE TITLE
RPM package depends on the daemonize package which is not available in CentOS 7 / RHEL 7 and EPEL 7

### ISSUE THREAD

**Issue by comolosabia** [aphyr/riemann#504]
Error: Package: riemann-0.2.8-1.noarch (/riemann-0.2.8-1.noarch)
           Requires: daemonize >= 1.7.3

**Comment by comolosabia** [comment]
Ups.. Duplicate of
https://github.com/aphyr/riemann/issues/453
